SpecRunner.html

```diff
@@ -1,23 +1,26 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Recursion Test Suite</title>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <link rel="stylesheet" href="lib/css/mocha.css" />
+<html lang="en">
 
-    <script src="lib/jquery.js"></script>
-    <script src="lib/chai.js"></script>
-    <script src="lib/mocha.js"></script>
-    <script src="lib/sinon.js"></script>
-    <script src="lib/testSupport.js"></script>
+<head>
+  <title>Recursion Test Suite</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <link rel="stylesheet" href="lib/css/mocha.css" />
 
-    <script src="src/recursion.js"></script>
+  <script src="lib/jquery.js"></script>
+  <script src="lib/chai.js"></script>
+  <script src="lib/mocha.js"></script>
+  <script src="lib/sinon.js"></script>
+  <script src="lib/testSupport.js"></script>
 
-    <script src="spec/part1.js"></script>
-    <script src="spec/part2.js"></script>
+  <script src="src/recursion.js"></script>
+
+  <script src="spec/part1.js"></script>
+  <script src="spec/part2.js"></script>
+
+</head>
+
+<body>
+  <div id="mocha"></div>
+</body>
```

Comment:
Since `head` and `body` are children of `html`, these should be indented

Reply:
I'm guessing that my linter is just trying to avoid an unnecessary nest on the entire file!